Makefile

```diff
@@ -133,7 +127,9 @@ hypervisor:
 		$(MAKE) -C $(T)/misc/efi-stub HV_OBJDIR=$(HV_OUT) EFI_OBJDIR=$(HV_OUT)/$(EFI_OUT); \
 	fi
 #endif
-
+	@if [ -f $(GENED_ACPI_INFO_HEADER) ] && [ "$(CONFIG_XML_ENABLED)" != "true" ]; then \
```

Comment:
I am not sure if this is a noticeable hint for users, better than the previous version though.
If the user make hypervisor and device model together, it is also easy to be ignored.